presto-main/src/main/java/io/prestosql/server/security/AuthenticationFilter.java

```diff
@@ -76,7 +78,8 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
         if (internalAuthenticationManager.isInternalRequest(request)) {
             Principal principal = internalAuthenticationManager.authenticateInternalRequest(request);
             if (principal == null) {
-                response.sendError(SC_UNAUTHORIZED);
+                response.setStatus(SC_UNAUTHORIZED);
+                response.setContentType(PLAIN_TEXT_UTF_8.toString());
```

Comment:
there is no content so no point in setting content type, is there any?

Reply:
Perhaps not, but seems better to be consistent. The response still has a body (of length zero), so it might be required to have a content type.